packages/web-components/fast-foundation/src/avatar/avatar.template.ts

```diff
@@ -0,0 +1,31 @@
+import { html, when } from "@microsoft/fast-element";
+import { Avatar } from "./avatar";
+
+/**
+ * The template for {@link @microsoft/fast-foundation#Avatar} component.
+ * @public
+ */
+export const AvatarTemplate = html<Avatar>`
+  <template>
+    <div 
+      class="coin ${x => (x.shape === "square" ? "square" : "circle")}"
```

Comment:
We may want to use the name of "control" here (vs the :host) - looks like we need a CSS part name here as well.

Reply:
> We may want to use the name of "control" here (vs the :host)

Could elaborate on this for me?

> \- looks like we need a CSS part name here as well

I'm not 100% sure what you mean by "CSS part name". Could you define that?

Reply:
Ah, I think found the definition. You're saying each element needs a corresponding `part` attributed. Is that correct?

Reply:
Yeah, we need a CSS Part to provide the opportunity to reach into the shadow DOM when supported by the browser.